static/docs/commands-reference/metrics_remove.md

```diff
@@ -76,13 +76,12 @@ Saving information to 'metrics.tsv.dvc'.
 Let's check the same DVC-file now:
 
 ```yaml
+md5: 7eca44a876f57f7a1614bea457876db2
 cmd: echo -e 'time/tauc/n2019-02-13/t0.9643' > metrics.tsv
-md5: 6f910c9000bb03492d1e66035ba8faf6
 outs:
-  - cache: false
-    md5: 7ce0bc12da7f88c1493763cdd4c3f684
-    metric: null
+  - md5: 9c012f709cfca85103d2680acfa5f628
     path: metrics.tsv
+    metric:
```

Comment:
??

Reply:
echo -e is not interpreted correctly in Windows. using GitBash

Reply:
so, why `mertic` parameter is empty?